src/signin.js

```diff
@@ -0,0 +1,41 @@
+import getConfig from "./config";
+import { connect, keyStores, WalletConnection } from "near-api-js";
+
+class ApiManager {
```

Comment:
we should make the file name match the export here, i'd think

Reply:
done